src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj

```diff
@@ -34,6 +34,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>2.2.0-preview</Version>
```

Comment:
@begoldsm let's continue using <versionPrefix> as that is being used across, but if you want to use <version> then you don't need <versionPrefix>. Also you do not need any package related properties.
All you need is the PackageId, assemblyName, assembly description and package tags. All the remaining ones not needed.
Also remove all commented code.